lib/ansible/modules/cloud/azure/azure_rm_virtualnetwork_facts.py

```diff
@@ -181,7 +181,8 @@ def __init__(self):
 
         self.results = dict(
             changed=False,
-            ansible_facts=dict(azure_virtualnetworks=[], virtualnetworks=[])
+            ansible_facts=dict(azure_virtualnetworks=[]),
```

Comment:
docs update?

Reply:
it is in the top level of the doc. No need to udpated.

Reply:
seems i looked at partial change...